doc/user-guides/acrn-dm-parameters.rst

```diff
@@ -469,10 +456,19 @@ arguments used for configuration.  Here is a table describing these emulated dev
 
    * - ``virtio-net``
      - Virtio network type device, parameter should be appended with the format:
-       ``virtio-net,<device_name>,[vhost],[mac=<XX:XX:XX:XX:XX:XX>]``.
-       The ``mac`` address is optional, ``device_name`` is the name of the TAP (or MacVTap) device.
-       It must include the keyword ``tap``.
+       ``virtio-net,<device_type=name>,[vhost],[mac=<XX:XX:XX:XX:XX:XX>],[mac_seed=<seed_string>]``.
+       The ``mac`` address is optional, ``name`` is the name of the TAP (or MacVTap) device.
        ``vhost`` specifies vhost backend, otherwise the VBSU backend is used.
```

Comment:
@weichenli-intel The commit message says "The final format is: `-s 4,virtio-net,tap/vmnet=dev_name,mac_seed=XXXX`". However, I don't see vmnet mentioned anywhere in the changes. Need to add description of vmnet?

Reply:
I have search current doc.
It's strange there was no mentioned of vmnet.
So I need check why and whether we should add it.

Reply:
Let's keep the silent of vmnet.